src/systemdhook.c

```diff
@@ -854,6 +854,38 @@ int main(int argc, char *argv[])
 		return EXIT_FAILURE;
 	}
 
+        const char *envs[] = {"process", "env", (const char *)0 };
+        yajl_val v_envs = yajl_tree_get(config_node, envs, yajl_t_array);
+        if (v_envs) {
+                for (unsigned int i = 0; i < YAJL_GET_ARRAY(v_envs)->len; i++) {
+                        yajl_val v_env = YAJL_GET_ARRAY(v_envs)->values[i];
+                        char *str = YAJL_GET_STRING(v_env);
+			/****
+ 			* If the oci-systemd-hook variable is passed with "disabled", 
+ 			* stop execution of oci-systemd-hook.
+ 			*****/
+                        if (strncmp (str, "oci-systemd-hook=", strlen ("oci-systemd-hook=")) == 0
+                        ||  strncmp (str, "oci-systemd-hook =", strlen ("oci-systemd-hook =")) == 0) {
```

Comment:
I thought it would fail if there was a space after the equals sign here if I didn't include the check.  But testing this morning proved that docker picked it off as an error well before I get to this point so I've removed the 'or' part of this statement.  Good thought.